### ISSUE TITLE
Resolve new issues reported by Pylint 2.13

### ISSUE THREAD

**Issue by andy-maier** [pywbem/nocaselist#79]
From test run https://github.com/pywbem/nocaselist/runs/5707314814?check_suite_focus=true#step:13:15

```
************* Module setup
setup.py:12:0: W0402: Uses of a deprecated module 'distutils' (deprecated-module)
```

**Comment by andy-maier** [comment]
Fixed by commit  e4b4499 (mistakenly pushed directly to master)

**Comment by andy-maier** [comment]
Rollback is not needed, since 1.0.5 will be released now.
